Content.Server/Backmen/Surgery/SurgerySystem.cs

```diff
@@ -190,7 +190,8 @@ private void OnStepAffixPartComplete(Entity<SurgeryStepAffixPartEffectComponent>
             RaiseLocalEvent(targetPart.Id, ref ev);
             // This is basically an equalizer, severing a part will badly damage it.
             // and affixing it will heal it a bit if its not too badly damaged.
-            _body.TryChangeIntegrity(targetPart, targetPart.Component.Integrity - BodyPartComponent.IntegrityAffixPart, false,
+            var healing = _body.GetHealingSpecifier(targetPart.Component) * 2;
```

Comment:
I don't understand this idea with "equalizer" too much, because in the past this just healed (or damaged???) the part by a bit. Now it just heals the part, without any fancy calculations.

Please add your idea in the new commit after porting this, I just have no ideas